apps/backend/prisma/migrations/20260729120000_oral_attendance/migration.sql

```
@@ -0,0 +1,12 @@
+CREATE TYPE "EventAttendanceStatus" AS ENUM ('PRESENT', 'ABSENT');
+
+ALTER TYPE "AttendanceCreationMethod" ADD VALUE 'ORAL_CALL' BEFORE 'SCANNER';
+
+ALTER TABLE "event_attendances"
+ADD COLUMN "status" "EventAttendanceStatus" NOT NULL DEFAULT 'PRESENT';
+
+CREATE INDEX "event_attendances_eventId_status_idx"
+ON "event_attendances"("eventId", "status");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)